01-node-tutorial/answers/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
does it make sense what is going on here in terms of package management?

Reply:
Yes, it makes sense. Here's what's going on:

- **test**: Runs a test command which currently just echoes an error message and exits with a non-zero status.
- **dev**: Runs `nodemon prompter.js` for development. `nodemon` will automatically restart the application when file changes are detected.
- **start**: Runs `node prompter.js` to start our application.